datadog/resource_datadog_downtime.go

```diff
@@ -305,6 +326,8 @@ func resourceDatadogDowntimeUpdate(d *schema.ResourceData, meta interface{}) err
 	if err = client.UpdateDowntime(dt); err != nil {
 		return fmt.Errorf("error updating downtime: %s", err.Error())
 	}
+	// handle the case when a downtime is replaced
+	d.SetId(strconv.Itoa(dt.GetId()))
```

Comment:
Just to confirm, the idea here being that we would be updating the current downtime on each subsequent apply, rather than creating a new one?

Reply:
It just ensures the downtime ID is set. In the future when downtimes are immutable this ID can change on updates